blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -66,17 +76,56 @@ const HeaderAccountAction = ({ customFields }) => {
 
   if (user && !error) {
     return (
-      <div className="xpmedia-subs-header">
-        <p>{user.displayName}</p>
-        <button type="button" onClick={handleLogout} className="xpmedia-subs-header--button">Log Out</button>
+      <div className="xpmedia-subs-header--header">
+        <div className="xpmedia-subs-header--desktop-header">
+          <Button
+            aria-expanded={isAccountMenuOpen}
+            as="button"
+            buttonSize={BUTTON_SIZES.SMALL}
+            buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+            buttonType={BUTTON_TYPES.LABEL_AND_TWO_ICONS}
+            iconType="user"
+            secondaryIconType={isAccountMenuOpen ? 'chevron-up' : 'chevron-down'}
+            onClick={() => setAccountMenu(!isAccountMenuOpen)}
+            text={phrases.t('identity-block.account')}
+            type="button"
+          />
+        </div>
+        <div className="xpmedia-subs-header--mobile-header">
+          <Button
+            aria-expanded={isAccountMenuOpen}
+            as="button"
+            buttonSize={BUTTON_SIZES.SMALL}
+            buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+            buttonType={BUTTON_TYPES.ICON_ONLY}
+            iconType="user"
+            onClick={() => setAccountMenu(!isAccountMenuOpen)}
+            text={phrases.t('identity-block.login-options')}
+            type="button"
+          />
+        </div>
+        {isAccountMenuOpen && (
+          <ul className="xpmedia-subs-header-dropdown--open">
+            <DropDownLinkListItem
+              href={manageAccountURL}
+              text={phrases.t('identity-block.manage-account')}
+            />
+            <DropDownLinkListItem
+              href={logoutURL}
+              onClick={handleLogout}
```

Comment:
is this onClick for a button element?

Reply:
The onClick is to fire the subs API event alongside the link href

Reply:
Just the native event... though, now that I'lm looking at it, why are we not using `onKeyUp` for the other one?  I thought general consensus was that you trigger events on the `exit` event.

Reply:
There is an `onKeyDown` to handle the keyboard events. Not sure I follow what you mean by exit event?

Reply:
feel like I'm missing something. right now, clicking the a tag does not log the user out in my testing. This is what I'm talking about https://github.com/WPMedia/fusion-news-theme-blocks/pull/1081/files#diff-934a642cc1ac4ed8dd1c0169c2affe1fff4ad5b42ef37bb9ecfee33a821f7d01R60

Reply:
It's more specifically for handling mouse events, but you listen to the onMouseUp/onKeyUp so that the user can "cancel" the event by hovering off the button or tabbing off the button to change their mind.  It also doesn't change the page from under their action.  Like, imagine clicking and holding (or long key pressing) on an image, it's replaced by another image, and now that image receives the mouse/key up event.

Reply:
I'm checking if the user is logged out by looking at the local storage. if logged out, it should be empty. 

<img width="985" alt="Screen Shot 2021-09-13 at 14 10 02" src="https://user-images.githubusercontent.com/5950956/133142390-b3d26752-2ec4-4d88-ae8d-f677e6239541.png">

<img width="609" alt="Screen Shot 2021-09-13 at 14 09 38" src="https://user-images.githubusercontent.com/5950956/133142338-eef482d1-d59b-4138-a5be-05fb10bfe512.png">

Reply:
Also, keydown can fire multiple times if held down.

Reply:
is clicking that a tag logging you out though? @matthewroach @nschubach like I said, when I check the local storage it's not clearing the authenticated user info. also the on click function is not firing because the element is not a button. 

I have not been using keyboard enter. When I removed the keyboard enter event listening, though, the function is still not firing because the element is a an a tag.

Reply:
ah it was because I was not passing props through